.dockerignore

```diff
@@ -0,0 +1,145 @@
+# WyzeSense2MQTT files
+config.yaml
+sensors.yaml
```

Comment:
The sample sensor.yaml should encourage people to quote the mac:
```yaml
"77890452":
  class: window
  name: A Window
```

Reply:
This one I'm a little more curious on. All the yaml information I've seen seem to encourage not quoting keys unless there are spaces or special characters. No problem doing this, just don't want to go against standard practice unless there is a good reason.

Reply:
Actually, I just remembered how it ended up that way. When moving to yaml, I created a conversion script that took a python dictionary and exported it as yaml, to make sure I got the structure right. That script exported without the quotes. Since it will automatically build (and rebuild) the sensors.yaml file every time a new sensor is detected, it would end up removing the quotes anyway as soon as someone adds or removes a sensor with it running. If they always did those actions with things stopped and manually edited the file, the quotes would stay, but otherwise they'll just vanish eventually.

Going to mark this as resolved.